iib/workers/tasks/build.py

```diff
@@ -1170,3 +1178,20 @@ def _adjust_csv_annotations(operator_csvs, package_name, organization):
             csv_annotations[annotation] = value
 
         operator_csv.dump()
+
+
+def _add_ocp_label_to_index(ocp_version, temp_dir, dockerfile_name):
```

Comment:
If I remember correctly, the functions in this file are alphabetically ordered. Can I please request you to move this function if what I mentioned is true?

Reply:
@yashvardhannanavati it looks like it used to be but @lcarva added a couple functions that are not, so I won't hold @shawn-hurley to that as well but it'd still be nice to do so.